packages/material-ui-lab/package.json

```diff
@@ -4,7 +4,11 @@
   "private": false,
   "author": "Material-UI Team",
   "description": "Laboratory for new Material-UI modules.",
-  "main": "./src/index.js",
+  "exports": {
+    ".": "./src/index.js",
+    "./es/*": "./src/*/index.js",
+    "./*": "./src/*/index.js"
```

Comment:
Would be nice but breaks down in packages where we mix `index.ts` and `index.js`. Since these are just files that define the public interface via `import`/`export` we should convert them to  TypeScript anyway. 

But there are some moving parts that need to be solved first. Considering we can't get rid of custom aliasing solutions anyway (see PR description) I'm closing until we can migrate completely.